src/utils/focus-management.ts

```diff
@@ -0,0 +1,183 @@
+// Credit: https://github.com/tailwindlabs/headlessui/blob/main/packages/%40headlessui-vue/src/utils/focus-management.ts
+import { dom } from '~/utils/dom'
+
+import type { Ref } from '@nuxtjs/composition-api'
+
+function getOwnerDocument<T extends Element | Ref<Element | null>>(
+  element: T | null | undefined
+) {
+  if (typeof window === 'undefined') return null
+  if (element instanceof Node) return element.ownerDocument
+  // eslint-disable-next-line no-prototype-builtins
+  if (element?.hasOwnProperty('value')) {
```

Comment:
We could follow the [advice from the rule](https://eslint.org/docs/rules/no-prototype-builtins) instead of disabling it, if you want:

```suggestion
  if (element && Object.prototype.hasOwnProperty.call(element, 'value')) {
```